### PR TITLE
Change subdivision code to iso standard full code 

### PR DESCRIPTION
**Pull request by rakris01**
full code is country code + "-" + state code.

Example:
California. CA will be US-CA

### CODE REVIEW

**Review by aniketgade01 — APPROVED** [review]

File: lib/countries/data/subdivisions/CN.yaml [hunk 1]
```diff
@@ -2057,7 +2057,7 @@ TW: #Code Changed in CLDR --
   name: Taiwan *
 #Removed from CLDR -- '91':
 #Removed from CLDR -- '92':
-HK:
```
Comment (davidejorgensen) [inline lib/countries/data/subdivisions/CN.yaml:2060]: @rakris01 Per our slack chat -- in CN.yaml why are there only updates for CN-HK and CN-MO but not CN-BJ nor others?

Comment (davidejorgensen) [reply lib/countries/data/subdivisions/CN.yaml:2060]: Verified Fixed now

File: lib/countries/data/subdivisions/FR.yaml [hunk 2]
```diff
@@ -6652,103 +6652,10 @@ PAC:
     zh: 普罗旺斯-阿尔卑斯-蓝色海岸
   geo:
   name: Provence-Alpes-Côte d'Azur
-6AE: #Code Changed in CLDR -- 
+FR-6AE: #Code Changed in CLDR -- 
   translations:
     en: Alsace
-#Removed from CLDR -- B:
```
[the remark this answers is not in the document]
Comment (davidejorgensen) [reply lib/countries/data/subdivisions/FR.yaml:6658]: Verified Fixed now.

**Comment by davidejorgensen** [comment]
Verified fix of legacy issues in BA.yaml (earlier PR's which commented out of the removals at the top level keyname of the subdivision, but without removing some of the child keys of each subdivision name, this left some invalid syntax in the files -- esp. repeating/duplicate but incomplete child keynames under the same parent).
Eg. the comment/removal of parent key '01' below should also remove the children 'translations' and 'en' keys as well. There were various of these across files... now confirmed fixed.
-----
#Removed from CLDR -- '01':
  translations:
    en: Brčko District
-----
Also confirmed per @rakris01 it's ok that the comments themselves are also removed.

**Comment by davidejorgensen** [comment]
Also confirmed per @rakris01 the change of syntax removing the single-quote (apostrophe) surrounding the subdivision code, eg. in AD.yaml the change of '08' --> AD-08 (no single quotes) is OK, has zero impact on any functionality nor the value that gets stored.

**Comment by davidejorgensen** [comment]
Per @rakris01 this PR covers the latest ISO updated subdivision codes and subdivision names, AND proper ISO code syntax (country code + dash + subdivision code). Verified this against previous change spreadsheet in https://coupadev.atlassian.net/browse/DOC-6964 and latest ISO.org online browsing platform (OBP) at https://www.iso.org/obp/ui/#iso:code:3166:CN and https://www.iso.org/obp/ui/#iso:code:3166:DK due to focus on these in JZ tickets originally driving the update.

**Comment by davidejorgensen** [comment]
LGTM

**Review by davidejorgensen — COMMENTED** [review]
Approved.

**Pull request merged**